images/nginx/rootfs/Dockerfile

```diff
@@ -71,8 +69,10 @@ RUN chmod +x /usr/local/entrypoint.sh \
   chown -R www-data.www-data ${dir}; \
   done'
 
-EXPOSE 80 443
+# enable sidecar / init cotnainer library path
+RUN echo "/lib:/usr/lib:/usr/local/lib:/modules_mount/usr/lib" > /etc/ld-musl-x86_64.path
```

Comment:
Hi Daniel, I had completely forgotten in the meeting yesterday so sorry for this late update.
I had moved he LD_LIBRARY_PATH variable from the base nginx image to the Controller Image. Please see this line https://github.com/kubernetes/ingress-nginx/blob/7806159b386bf34b3ef0d8f130e66ad0247903af/rootfs/Dockerfile#L34 . So let us move this step of creating the `/etc/ld-musl-x86_64.path` file, from here to the Controller Image. So kindly remove the line here for now. I will take care of adding the file to the Controller Image